packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts

```diff
@@ -35,9 +35,9 @@ const transformValidator = (
 })
 
 export abstract class CosmosSdkBaseAdapter<T extends CosmosChainTypes> implements IChainAdapter<T> {
-  protected readonly chainId: CAIP2
-  protected readonly assetId: CAIP19 // This is the caip19 for native token on the chain (ATOM/OSMO/etc)
-  protected readonly supportedChainIds: CAIP2[]
+  protected readonly chainId: ChainId | CAIP19
+  protected readonly assetId: AssetId | CAIP2 // This is the caip19 for native token on the chain (ATOM/OSMO/etc)
```

Comment:
```suggestion
  protected readonly assetId: AssetId | CAIP19 // This is the caip19 for native token on the chain (ATOM/OSMO/etc)
```

Reply:
Should we change the reference to caip19 in the comment as well?

Reply:
I think it is meant to be `caip19`, as it is currently.

Reply:
Yeah I meant changing the comment to `assetId` - I get it why we have both types in this PR, since there'll be a follow-up removing CAIP-XX references, but the comments can already be moved to assetId vs caip19 (if that's something we want)

Reply:
Ah yes, I'm with you - we should.

Not a huge issue if missed though, as it'll happen in a future PR in this epic.